### PR TITLE
MINOR: Remove magic number and extract Pattern instance from method as class field

### PR DESCRIPTION
**Pull request by asdf2014**
* Remove magic number
* Extract Pattern instance from method as class field
* Add `@Override` declare

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by asdf2014** [comment]
Hi, @guozhangwang . PTAL.

**Comment by guozhangwang** [comment]
cc @rhauch @kkonstantine @ewencp

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java [hunk 1]
```diff
@@ -109,6 +109,8 @@
 public class DistributedHerder extends AbstractHerder implements Runnable {
     private static final Logger log = LoggerFactory.getLogger(DistributedHerder.class);
 
+    private static final long FORWARD_REQUEST_SHUTDOWN_TIMEOUT_MS = 10_000L;
+    private static final long START_AND_STOP_SHUTDOWN_TIMEOUT_MS = 1000L;
```
Comment (rhauch) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java]: I'd prefer that we use `TimeUnit.SECONDS.toMillis(10)` and `TimeUnit.SECONDS.toMillis(1)` here. It avoids having to do any math when looking at the code.

**Review by rhauch — COMMENTED** [review]
@asdf2014 thanks for the fixes. One suggestion below.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java (hunk 1 above)
Comment (asdf2014) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java]: Thanks! It is better than pure number.

**Comment by asdf2014** [comment]
Hi, @rhauch @guozhangwang . Already using `TimeUnit` instead of those pure number. PTAL.

**Review by rhauch — APPROVED** [review]
Thanks! LGTM.

**Comment by asdf2014** [comment]
@rhauch Thx a lot.

**Pull request merged**